# This is a patch for Net-Whois-0.23/Whois.pm to update it to Net-Whois-0.23-ivan1/Whois.pm.
# It was generated by makepatch 1.93 (2.0BETA) on Tue Aug  3 02:40:13 1999.
#
# To apply this patch, chdir to source directory Net-Whois-0.23/Whois.pm and enter
#
#     patch -p1 -N < <this-file>

exit
# End of preamble.

# Patch input follows.
__DATA__
Index: Net-Whois-0.23-ivan1/Whois.pm
Prereq:  1.13 
#######  => 
*** Net-Whois-0.23/Whois.pm	Fri Jan 29 22:00:46 1999
--- Net-Whois-0.23-ivan1/Whois.pm	Tue Aug  3 02:33:57 1999
***************
*** 216,224 ****
      my %info;
  
      do {
! 	$t = shift @text;
!     } while (@text && ($t eq '' || $t =~ /^registrant:/i));
!     @info{'NAME', 'TAG'} = ($t =~ /^(.*)\s+\((\S+)\)$/)
  	or return;
  
      @t = ();
--- 216,225 ----
      my %info;
  
      do {
!       $t = shift @text;
!     } while ( @text && ($t !~ /^registrant:/i ) );
!     $t = shift @text or return;
!     @info{'NAME', 'TAG'} = ( $t =~ /^(.*)\s+\((\S+)\)$/)
  	or return;
  
      @t = ();


End of Patch.
